custom_components/tuya_local/devices/zyhps01_human_presence_sensor.yaml

```diff
@@ -0,0 +1,124 @@
+name: Human presence sensor
+products:
+  - id: wzp9yjsobanavsut
+    name: Human Presence Sensor
```

Comment:
This name should be the manufacturer (if known) and model (ZY-HPS-01?) of the product.